.github/workflows/main.yml

```diff
@@ -78,31 +78,39 @@ jobs:
       - name: Build
         run: cmake --build build
 
-      - name: Package
```

Comment:
So, we'll still build MSYS2 64-bit binaries as part of the CI process, but we don't upload artifacts anymore and they won't be part of the official release binaries (i.e. these will be vcpkg builds)?

Reply:
> So, we'll still build MSYS2 64-bit binaries as part of the CI process, but we don't upload artifacts anymore and they won't be part of the official release binaries (i.e. these will be vcpkg builds)?

Yes, that's the plan. I'm not sure about the `lukka/run-vcpkg@v10` action, maybe I should write the vcpkg caching script myself. So this PR is still at the experimental stage.